### PR TITLE
Blaze: Implement mini-carousel in /stats

### PR DESCRIPTION
**Pull request by candy02058912**
#### Proposed Changes

| Mobile  | Desktop |
| :-------------: | :-------------: |
|   | |

Folow up issues: 

#### Testing Instructions

* Go to `/stats/day/:site?flags=stats-mini-carousel`
* Check if it works well on desktop and mobile
* Check if track click events are the same as before

#### Pre-merge Checklist

- [x] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] ~[Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?~
- [x] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [ ] ~Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)~
- [x] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] ~For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?~

Fixes https://github.com/Automattic/dotcom-forge/issues/1460

### CODE REVIEW

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7730955

Thank you @candy02058912 for including a screenshot in the description! This is really helpful for our translators.

**Comment by DustyReagan** [comment]
Looks great on mobile and web! I have a couple questions about Tracks. Click tracking appears to be working, but do we need to use a different Tracks name for the mini-banner clicks to differentiate them from clicks on the banner at the bottom of the page, or are we intentionally consolidating those counts?

Also, should we be tracking mini-banner impressions?

**Comment by allilevine** [comment]
> Click tracking appears to be working, but do we need to use a different Tracks name for the mini-banner clicks to differentiate them from clicks on the banner at the bottom of the page, or are we intentionally consolidating those counts?

That's a good point, we probably want to be able to track them separately to see if the mini-banners perform better.

> Also, should we be tracking mini-banner impressions?

How do we track banner impressions now?

**Comment by candy02058912** [comment]
This PR is guarded with `?flags=stats-mini-carousel` because I'd like to focus on the UI, CTA, and string translations.

The mini-carousel is supposed to replace the bottom banner(https://github.com/Automattic/dotcom-forge/issues/1488)'s Yoast and Blaze, so that's why the same name was reused for the mini-carousel. Was following the description [here](https://github.com/Automattic/dotcom-forge/issues/1460).
> All track click events are the same as the bottom banners

I thought it made sense, so I didn't change the event name.

> Also, should we be tracking mini-banner impressions?

@DustyReagan @allilevine I was thinking of using what [the current code is using for the bottom banner](https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/stats/promo-cards/index.jsx#L40-L63), and it seems like a big chunk, so I broke this one out into another issue: https://github.com/Automattic/dotcom-forge/issues/1487

In short:
* This PR will not cover the impression events
* Discussion: Would it make sense to register a new event or reuse the old one (already in this PR)?

**Comment by allilevine** [comment]
> The mini-carousel is supposed to replace the bottom banner(https://github.com/Automattic/dotcom-forge/issues/1488 Yoast and Blaze, so that's why the same name was reused for the mini-carousel. Was following the description https://github.com/Automattic/dotcom-forge/issues/1460.

Got it, so because we're removing the bottom banner and adding the top banner at the same time, the tracking won’t overlap. Using the same event name seems fine in that case, since we'll know when we moved the banner.

@candy02058912 @DustyReagan If everything else looks good, and we're behind a feature flag, let's merge!

**Review by DustyReagan — APPROVED** [review]
Looks good! 👍

**Review by lupus2k — APPROVED** [review]
LGTM 🎉

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7730955).
